lib/cli.js

```diff
@@ -83,6 +83,13 @@ function cli()
 				info: "URL to an HTML document.",
 				type: require("url")
 			},
+			"host-requests":
+			{
+				rename: "maxSocketsPerHost",
+				type: Number,
```

Comment:
Move this line below `info:`, to be consistent with other options.

Reply:
Still incorrect order. See https://github.com/staabm/broken-link-checker/blob/05d24729df75e56612b4df1f15439ad5f3708062/lib/cli.js#L36-L41

Reply:
^^

Reply:
arghs ;). fixed.